points_selection_vispy.py

```diff
@@ -0,0 +1,266 @@
+"""
+    3D Points Picking & Selection tool based on Vispy library
+    Author: Penny Xuran Qian (NAOC/SAO) pennyqxr@gmail.com
+    Date: Feb 15, 2016
+    License: GPL
```

Comment:
@PennyQ note that Glue and the 3D viewers are released under the BSD license, so if you commit this code under the GPL, we won't be able to use it (GPL can't be used in BSD package). Could you update to BSD?